<launch>
    <node name="ypspur_ros" pkg="ypspur_ros" type="ypspur_ros" output="screen">
        <param name="port" value="/dev/serial/by-id/usb-T-frog_project_T-frog_Driver-if00"/>
        <!-- <param name="port" value="/dev/ttyACM0"/> -->
        <param name="param_file" value="/home/takeda-y/researches/programs/platform/yp-robot-params/robot-params/beego.param"/>
    </node>
   
    <node name="hokuyo" pkg="hokuyo_node" type="hokuyo_node" respawn="false" output="screen">
        <param name="calibrate_time" type="bool" value="false"/> 
        <param name="port" type="string" value="/dev/serial/by-id/usb-Hokuyo_Data_Flex_for_USB_URG-Series_USB_Driver-if00"/> 
        <param name="intensity" type="bool" value="false"/>
        <param name="angle_min" type="double" value="-0.7853"/> 
        <param name="angle_max" type="double" value="0.7853"/> 
    </node>
    <node name="kadai" pkg="yamasemi_kadai" type="kadai" output="screen"/>
</launch>
